src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.security.SecureRandom;
+
 public class ColorSupplier {
+    private static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
`Random` is an instance of another object, so it is our dependency and it cannot be constant, so let's not name it as it is, and make it non-static, we will learn how to work and initialize objects with dependencies later on
Constants can be primitives wrappers or literals